### PR TITLE
Add OpenTelemetry in Focus post for April

### PR DESCRIPTION
**Pull request by austinlparker**
I mean, the first day of the next month is pretty much like the last day of the prior month, right?

### CODE REVIEW

File: content/en/blog/2023/otel-in-focus-04.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+OpenTelemetry at KubeCon EU, which was a blast. Can't wait for Chicago this fall!
+
+Are you a maintainer with something you’d like featured here? Get in touch with
+me [via email](mailto:austin@lightstep.com), or on the
+[CNCF Slack #otel-comms](https://cloud-native.slack.com/archives/C02UN96HZH6)
+channel.
+
+## Releases and Updates
+
+Here are the latest updates from our core repositories.
```
Comment (cartermp) [inline content/en/blog/2023/otel-in-focus-04.md:20]:
```suggestion
Here are the latest updates from some of our core repositories.
```

File: content/en/blog/2023/otel-in-focus-04.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+has been released, and it's a big one!
+
+First, **OpenTelemetry Protocol** has been declared stable! Second, we've
+started a process to converge the [Elastic Common Schema with OpenTelemetry
+Semantic Conventions](/blog/2023/ecs-otel-semconv-convergence/). What does this
+mean? At a high level, you can expect to see that semantic conventions will
+split out of the specification as we proceed towards aligning our standards.
+Please be on the look out for more information.
+
+Other changes include -
```
Comment (cartermp) [inline content/en/blog/2023/otel-in-focus-04.md:34]:
```suggestion
Other changes include:
```

File: content/en/blog/2023/otel-in-focus-04.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+## Project Updates
+
+KubeCon EU saw over ten thousand cloud-native developers gather in Amsterdam,
+and a lot of you stopped by the OpenTelemetry booth to say hi! Hopefully some of
+you got your hands on our limited-edition KubeCon stickers... if not, well,
+there'll be more limited edition stickers. Just not for KubeCon, because it's
+come and gone.
+
+There was a lot of great feedback that we're excited to tackle as a project over
+the coming months, including --
```
Comment (cartermp) [inline content/en/blog/2023/otel-in-focus-04.md:86]:
```suggestion
the coming months, including:
```

**Review by cartermp — APPROVED** [review]

**Comment by austinlparker** [comment]
you know i love emdashes phil

**Comment by cartermp** [comment]
is that new because of your birthday or

**Comment by austinlparker** [comment]
i've always loved emdashes

**Pull request merged**